### PR TITLE
Updated DSA resources

### PR DESCRIPTION
**Pull request by simshadows**

### CODE REVIEW

**Comment by simshadows** [comment]
making a comment for discord webhook testing

**Comment by simshadows** [comment]
making a comment for discord webhook testing

**Comment by simshadows** [comment]
making a comment for discord webhook testing

**Comment by simshadows** [comment]
making a comment for discord webhook testing

**Comment by simshadows** [comment]
making a comment for discord webhook testing

**Comment by zax-xyz** [comment]
ratio

**Comment by dchenz** [comment]
hello world

**Comment by simshadows** [comment]
👋🏻

**Comment by dchenz** [comment]
hmm should probably disable comments lmao

File: src/pages/resources/data-structures-and-algorithms.mdx [hunk 1]
```diff
@@ -3,61 +3,89 @@ layout: ../../layouts/MDLayout.astro
...
 keywords: ["dsa", "cp", "competitive programming"]
-
-hidetoc: true
 ---
 
 This page is a compilation of *free* resources for people to use. The categories are taken from [neetcode](https://neetcode.io/practice) since we're currently (as of writing) following this study plan as a community.
 
 ## General Resources
 
+Here's a bunch of different resources to check out
```
Comment (zax-xyz) [inline src/pages/resources/data-structures-and-algorithms.mdx:12]: ratio

**Review by zax-xyz — COMMENTED** [review]
ratio

File: src/pages/resources/data-structures-and-algorithms.mdx (hunk 1 above)
Comment (simshadows) [reply src/pages/resources/data-structures-and-algorithms.mdx:12]: ratio

**Comment by simshadows** [comment]
making a comment for discord webhook testing

File: src/pages/resources/data-structures-and-algorithms.mdx (hunk 1 above)
Comment (simshadows) [reply src/pages/resources/data-structures-and-algorithms.mdx:12]: making a comment for discord webhook testing

Comment (simshadows) [reply src/pages/resources/data-structures-and-algorithms.mdx:12]: i am once again making a comment for discord webhook testing

**Pull request merged**